### PR TITLE
Update lob-apps-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the ignore app version segment to reflect that it supports windows apps with self-updating installers(such as Google Chrome).

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2312

File: memdocs/intune/apps/lob-apps-windows.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ A line-of-business (LOB) app is one that you add from an app installation file.
     - **Description**: Enter the description of the app. The description appears in the company portal.
     - **Publisher**: Enter the name of the publisher of the app.
     - **App Install Context**: Select the install context to be associated with this app. For dual mode apps, select the desired context for this app. For all other apps, this is pre-selected based on the package and cannot be modified.
-    - **Ignore app version**: Set to **Yes** if the app developer automatically updates the app. This option applies to mobile .msi apps only.
+    - **Ignore app version**: Set to **Yes** if the app developer automatically updates the app. This option applies to mobile .msi apps and windows apps with self-updating installers(such as Google Chrome).
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/lob-apps-windows.md:71]:
```suggestion
    - **Ignore app version**: Set to **Yes** if the app developer automatically updates the app. This option applies to mobile .msi apps and Windows apps with self-updating installers (such as Google Chrome).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder. Thnaks

**Pull request merged**